package.json

```diff
@@ -138,12 +138,25 @@
       "common-def": "common/lib/common-definitions",
       "common-utils": "common/lib/common-utils",
       "protocol-def": "common/lib/protocol-definitions",
-      "azure": "azure",
+      "azure": {
```

Comment:
It seems somewhat odd that we would define this here, and not in "azure" directory since it is its own Monorepo. I understand that root package.json can manage config across all monorepos (not just client), but when developing around azure-client I do spend most of the time in azure folder, (likely wrongly) assuming that its fully self contained.

Reply:
@Josmithr had the same feedback. I'll explore that option this morning.